AutoCollection/WebSnippet.ts

```diff
@@ -47,9 +47,13 @@ class WebSnippet {
       
     }
 
-    public enable(isEnabled: boolean) {
+    public enable(isEnabled: boolean, webSnippetInstrumentationKey?: string ) {
         this._isEnabled = isEnabled;
-
+        
+        if (webSnippetInstrumentationKey) {
+            WebSnippet._snippet = snippetInjectionHelper.webSnippet.replace("INSTRUMENTATION_KEY", webSnippetInstrumentationKey);
```

Comment:
You already replaced this value in the constructor right?, is this code working?

Reply:
Oh I see, you replace the original one that was not updated before "snippetInjectionHelper.webSnippet"